Gruntfile.js

```diff
@@ -739,6 +739,11 @@ module.exports = function(grunt) {
       },
     },
     protractor: {
+      'release-e2e-tests':{
+        options:{
+          configFile: 'tests/relase.tests.conf.js'
```

Comment:
There's a typo in the config file name.

Also, I think it should be called `regression-tests.conf.js` because I want to run it every build, not just pre-release. If it ends up taking way too long then we can change it up but for the near future we should run it as early as possible.

Thirdly, I don't think this should be a separate suite to the other e2e test suite - they're all regression tests. Right now it sort of has to be because we're using different config, but are you envisioning one day all tests will run against the standard config?